test-partner/Dockerfile.debug-partner

```diff
@@ -2,7 +2,7 @@ FROM registry.access.redhat.com/ubi8/ubi:latest
 
 RUN \
     yum -y update && \
-    yum -y install pciutils util-linux net-tools procps-ng iputils iproute jq libselinux-utils && \
+    yum -y install pciutils util-linux net-tools procps-ng iputils iproute jq libselinux-utils iptables nftables&& \
```

Comment:
Could you add a space here after `nftables` and before the `&&`?

Reply:
fixed